feat(changelog): jobs-added-serverless-jobs-max-timeout-increased-to-24h 2024-05-14 (#3170)
scaleway/docs-content: changelog/may2024/2024-05-14-jobs-added-serverless-jobs-max-timeout-increased-to-24h.mdx

---
title: Serverless Jobs max timeout increased to 24h
status: added
author:
    fullname: 'Join the #serverless-jobs channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-05-14
category: serverless
product: jobs
---

> Long running Jobs are welcome on our infrastructure ! Max timeout is now of 24 hours instead of 6 hours

Reviewer: ```suggestion
Long running Jobs are welcome on our infrastructure! Max timeout has now increased to 24 hours, from the previous limit of 6 hours.
```
